app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -328,6 +329,10 @@ public void onClick(View v) {
             }
         });
         etSearchStream = (EditText) findViewById(R.id.stream_drawer_search);
+        if (isCurrentThemeNight) {
+            etSearchPeople.setTextColor(ContextCompat.getColor(this, R.color.colorPrimary));
```

Comment:
I was wondering why color primary. Can we use Color.WHITE here? 
If we change the primary color of our app in future, this may cause some trouble.